pkg/users/accounts-list.js

```diff
@@ -324,6 +394,22 @@
 
         return true;
     });
+    const ref = useRef();
+
+    useEffect(() => {
+        if (ref.current != currentTextFilter) {
+            ref.current = currentTextFilter;
+            const newOptions = { ...options };
+            if (currentTextFilter) {
+                newOptions.user = currentTextFilter;
+            } else {
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230119-104710-24ec9d00-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)